geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/ingest/ScaldingConverterIngestJob.scala

```diff
@@ -0,0 +1,119 @@
+/***********************************************************************
+* Copyright (c) 2013-2015 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0 which
+* accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+package org.locationtech.geomesa.tools.ingest
+
+import java.net.URLDecoder
+import java.nio.charset.StandardCharsets
+
+import com.twitter.scalding.{Args, Hdfs, Job, Local, Mode}
+import com.typesafe.config.ConfigFactory
+import com.typesafe.scalalogging.slf4j.Logging
+import org.geotools.data.{DataStoreFinder, Transaction}
+import org.geotools.filter.identity.FeatureIdImpl
+import org.locationtech.geomesa.accumulo.data.AccumuloDataStore
+import org.locationtech.geomesa.accumulo.data.AccumuloDataStoreFactory.{params => dsp}
+import org.locationtech.geomesa.accumulo.index.Constants
+import org.locationtech.geomesa.convert.SimpleFeatureConverters
+import org.locationtech.geomesa.convert.Transformers.DefaultCounter
+import org.locationtech.geomesa.jobs.scalding.MultipleUsefulTextLineFiles
+import org.locationtech.geomesa.tools.Utils.IngestParams
+import org.locationtech.geomesa.utils.geotools.SimpleFeatureTypes
+
+class ScaldingConverterIngestJob(args: Args) extends Job(args) with Logging {
+  import scala.collection.JavaConversions._
+
+  val counter = new DefaultCounter
+
+  lazy val pathList         = DelimitedIngest.decodeFileList(args(IngestParams.FILE_PATH))
+  lazy val sftSpec          = URLDecoder.decode(args(IngestParams.SFT_SPEC), StandardCharsets.UTF_8.displayName)
```

Comment:
not sure. i carried that over from previous code. i assume that the scalding stuff does some sort of argparse that required it...since it treats args like --argname value i'll see if i can figure out exactly why and comment it